roles/tendrl-performance-monitoring/tasks/binary.yml

```diff
@@ -0,0 +1,5 @@
+# TODO Add when packages are ready
+#- name: Install tendrl-performance-monitoring
+#  yum:
+#    name=tendrl-performance-monitoring
+#    state=latest
```

Comment:
I guess that this is commented out because there is no build for `tendrl-performance-monitoring` yet, but I would rather keep it there so that when someone runs it and the package is still not there, ansbile run would fail with a clear error message. When I run this role with packages (which is the default) now, it would say "ok" - which is kind of misleading.